app/views/docs/functions.phtml

```diff
@@ -384,6 +384,35 @@ func main(req: RequestValue, res: RequestResponse) throws -> RequestResponse {
     <p>If you're running Appwrite locally and trying to connect to your local Appwrite instance from a cloud function, you'll have to use your local IP address in place of localhost. This is because, within the container localhost does not refer to the host machine rather the container itself. </p>
 </div>
 
+<h2><a href="/docs/functions#ignoreFiles" id="ignoreFiles">Ignore Files</a></h2>
+
+<p>Library folders such as <b>node_modules</b> or <b>vendor</b> should not be included in the tarball, although you will want them locally so you have syntax highlighting in your IDE. Similarly, you should <b>not</b> include files containing secrets in your deployment. You can use the Appwite CLI's <b>file ignoring</b> feature to exclude specific files from a deployment.</p>
+
+<p>You can open the <b>appwrite.json</b> file and look for your function's definition. In there, you can set the value for the <b>ignore</b> key to specify which files and folders should be ignored. This value must be an array, as seen in the example below:</p>
+
+<div class="ide margin-bottom" data-lang="json" data-lang-label="JSON">
```

Comment:
Not sure "json" is a language and if prism supports it. I think we should use Javascript instead

Reply:
There is JSON in other places such as models, CLI or Realtime. Example. https://appwrite.io/docs/realtime#payload

Should I replace it anyway?

Reply:
Okay if syntax highlighting is working, we're good 👍